l3kernel/expl3.dtx

```diff
@@ -771,7 +774,7 @@
 % \end{verbatim}
```

Comment:
line above uses the wrong variant string, should be `cne` to reflect the changes in the text above.

Reply:
Corrected